src/components/ControllerTop.vue

```diff
@@ -26,7 +29,10 @@
         ></v-select>
       </div>
       <div class="topctrl-layouts">
-        <label id="drop-label-version" class="drop-label"
+        <label
+          id="drop-label-version"
+          v-tooltip.top-end="`${Object.keys(layouts).length} layouts`"
+          class="drop-label"
           >{{ $t('layout.label') }}:</label
```

Comment:
There's two different closing HTML tags approaches


```html
<elem
  prop1
  prop2
  >.....</elem
>
```

and

```html
<elem
  prop1
  prop2
>
  .....
</elem>
```

I think this should be unified.

Reply:
This is auto formatted by prettier. The reason it does this is not introduce extra spaces which provide split points I believe.

Reply:
you added spaces, which was not the intention.